integrated_channels/sap_success_factors/exporters/course_metadata.py

```diff
@@ -109,13 +110,20 @@ def transform_status(self, course_run):
             else self.STATUS_INACTIVE
         )
 
-    def transform_title(self, course_run):
+    def get_title(self, course_run):
         """
-        Return the transformed version of the course title, as well as the locale.
+        Return the transformed version of the course title, as well as the locale. For all instructor-paced courses
+        also include the start date to distinguish multiple runs of the same course (ENT-782)
         """
+        title = course_run.get('title') or ''
+        if course_run.get('pacing_type') == 'instructor_paced' and course_run.get('start'):
+            title += ' (Starts: {:%B %Y})'.format(parse_lms_api_datetime(course_run.get('start')))
+        return title
```

Comment:
This is the same code as what we have in the Degreed integration. Is there some way to move this to a common base class?

Reply:
Also, maybe rename the shared function to something like `format_title`.